src/WebApp/Controllers/CustomerManagementController.cs

```diff
@@ -1,10 +1,13 @@
+using System.Diagnostics;
+
 namespace PitStop.WebApp.Controllers;
 
 public class CustomerManagementController : Controller
 {
     private readonly ICustomerManagementAPI _customerManagementAPI;
     private readonly Microsoft.Extensions.Logging.ILogger _logger;
     private ResiliencyHelper _resiliencyHelper;
+    private readonly ActivitySource _activitySource = new ActivitySource("test-source");
```

Comment:
Test-source in 'productie' code?

Reply:
Ik herinner eerlijk gezegd niet dat ik dit toegevoegd heb, ik heb verder ook niet gewerkt met customermanagament